buildSrc/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-instrumentation.gradle.kts

```diff
@@ -1,10 +1,15 @@
 plugins {
   id("io.opentelemetry.instrumentation.javaagent-testing")
   id("io.opentelemetry.instrumentation.muzzle-check")
+  id("io.opentelemetry.instrumentation.javaagent-codegen")
 }
 
 dependencies {
   add("muzzleBootstrap", "io.opentelemetry.javaagent:opentelemetry-javaagent-bootstrap")
   add("muzzleTooling", "io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
   add("muzzleTooling", "io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api")
+
+  add("codegen", "io.opentelemetry.javaagent:opentelemetry-javaagent-bootstrap")
+  add("codegen", "io.opentelemetry.javaagent:opentelemetry-javaagent-tooling")
```

Comment:
Is the `codegen` configuration still needed? The codegen plugin should only require the `extension-api` module, is it possible for it to depend on it directly?

Reply:
Removed